### PR TITLE
feat: add option to configure commit- rules

### PR DESCRIPTION
**Pull request by dolby360**
Signed-off-by: dolby360 <mrdolbyking@gmail.com>

## Description
Add option to configure commit rules

**Related issue (if any):** fixes #*issue_number_goes_here*

## Checklist

- [v] I have followed this repository's contributing guidelines.
- [v] I have used the conventional commits specification for my commit messages.
- [v] I have signed my commits.
- [v] I will adhere to the project's code of conduct.

### CODE REVIEW

**Review by TomerFi — DISMISSED** [review]
@dolby360 🔥

File: .github/auto-me-bot.yml [hunk 1]
```diff
@@ -2,5 +2,6 @@
 ---
 pr:
   conventionalCommits:
+    'body-max-line-length': [2, 'always', 150]
```
Comment (TomerFi) [inline .github/auto-me-bot.yml:5]: Maybe this should be under the `rules` directive and not directly under the `conventionalCommits` directive?

That way we can keep some kind of correlation to the actual [configuration object](https://commitlint.js.org/#/reference-configuration?id=javascript), so we can perhaps more easily add more configuration values in the future like `parserPreset`, `formatter`, and etc.

https://github.com/TomerFi/auto-me-bot/issues/45#issuecomment-1200277038

**Review by TomerFi — CHANGES_REQUESTED** [review]

**Review by TomerFi — APPROVED** [review]
❤️

**Comment by TomerFi** [comment]
@all-contributors please add @dolby360 for code and doc

**Comment by TomerFi** [comment]
@all-contributors please add @dolby360 for code and doc

**Comment by TomerFi** [comment]
@dolby360 
It's the second time if not more we're collaborating.
you're good - pleasure working with you!

Thank you for your contribution which is included in the new release [1.1.0](https://github.com/TomerFi/auto-me-bot/releases/tag/1.1.0)

**Comment by dolby360** [comment]
> @dolby360 It's the second time if not more we're collaborating. you're good - pleasure working with you!
> 
> Thank you for your contribution which is included in the new release [1.1.0](https://github.com/TomerFi/auto-me-bot/releases/tag/1.1.0)

Sure I'm trying to get into open source for a while and your kind approach and responsiveness got me hooked on your projects.

**Pull request merged**